### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by vikeri** [victorliun/trega#1]
Seems like just what I'm looking for. But could you please list in the readme how to use this for someone not too familiar with python-land?

**Comment by lootsy** [comment]
I don't think that the current code works out of the box. Therefore updating the readme isn't the only to-do.
